apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1480,11 +1480,12 @@ naming_system_manage_name(_Config) ->
 
     {ok, 200, #{<<"balance">> := Balance3}} = get_balance(),
     Host = internal_address(),
-    http_request(Host, post, "spend-tx",
-        #{recipient_pubkey => Name,
-          amount => 77,
-          fee => 50}),
+    {ok, 200, _} = http_request(Host, post, "spend-tx",
+                                #{recipient_pubkey => Name,
+                                  amount => 77,
+                                  fee => 50}),
     aecore_suite_utils:mine_blocks(aecore_suite_utils:node_name(?NODE), 1),
```

Comment:
This function actually returns the mined block. Test could be improved further by checking that returned mined block has the expected tx.